lib/MusicBrainz/Server/Controller/User.pm

```diff
@@ -440,7 +440,7 @@ sub tag : Chained('load') PathPart('tag') Args(1)
     my $tag = $c->model('Tag')->get_by_name($tag_name);
     my %tags = ();
     my $tag_in_use = 0;
-    my @entities_with_tags = sort { $a <=> $b } entities_with('tags');
+    my @entities_with_tags = sort { $a cmp $b } entities_with('tags');
```

Comment:
Yes, the order will be independent of the interface language. It’s the result of sorting the internal (English) type names, so it will be `area`, `artist`, `instrument`, `label`, `place`, `recording`, `release`, `release_group`, `series`, `work` in that order.